gradle-versions-plugin/src/main/kotlin/com/github/benmanes/gradle/versions/updates/DependencyUpdatesTask.kt

```diff
@@ -12,6 +14,7 @@ import org.gradle.api.tasks.Internal
 import org.gradle.api.tasks.Optional
 import org.gradle.api.tasks.TaskAction
 import org.gradle.util.ConfigureUtil
+import java.lang.IllegalArgumentException
```

Comment:
This import should not be needed in Kotlin.

Reply:
That's right, removed!